tester/runner.go

```diff
@@ -574,3 +574,15 @@ func LoadBundles(args []string, filter loader.Filter) (map[string]*bundle.Bundle
 
 	return bundles, nil
 }
+
+//AggregateBundleModules will retrieve modules from bundles
```

Comment:
Unless there's a good reason to export this function from the tester package, let's move it into the cmd package where it's used an unexport it. Fewer exported functions is better.

Reply:
Modified code as per recommendation - no function to export now from tester package